eclipse-theia/theia-blueprint
# [Java] Pin `vscjava.vscode-java-dependency`

#### What it does

The latest version published on open-vsx.org is faulty [[1]]. So, we temporarily pin that extension to the latest one know to work: `0.21.2`, as per issue below. See also this related discussion on the Theia repo [[2]]

[1]: https://github.com/microsoft/vscode-java-dependency/issues/763
[2]: https://github.com/eclipse-theia/theia/discussions/12610

#### How to test

Check that the expected version of "`vscjava.vscode-java-dependency`" is downloaded and make sure the Java Projects view works. i.e. the "java Projects" view should be populated when a Java project is opened:

![image](https://github.com/eclipse-theia/theia-blueprint/assets/25749063/5b136972-1351-4ab7-9ce8-5cdd022607b6)

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
